docs/next.config.js

```diff
@@ -72,7 +72,7 @@ export default withNextra({
       {
         source: "/:path(.*)",
         has: [{ type: "host", value: "warnings.authjs.dev" }],
-        destination: "https://authjs.dev/reference/warnings/:path*",
+        destination: "https://authjs.dev/reference/core/errors#warningcode",
```

Comment:
TBD: should we put warning code under /errors or we make a new file for it in the source code.